CppMicroServicesConfig.cmake.in

```diff
@@ -87,12 +87,9 @@ set(US_LIBRARIES )
 set(US_RUNTIME_LIBRARY_DIRS )
 
 # Components support
-set(US_BUNDLES Framework HttpService WebConsole ShellService DeclarativeServices) # ConfigAdmin EventAdmin ...
+set(US_BUNDLES Framework DeclarativeServices) # ConfigAdmin EventAdmin ...
 
 set(US_Framework_BUNDLE_DEPS )
-set(US_HttpService_BUNDLE_DEPS Framework)
-set(US_WebConsole_BUNDLE_DEPS ${US_HttpService_BUNDLE_DEPS} HttpService)
-set(US_ShellService_BUNDLE_DEPS Framework)
 set(US_DeclarativeServices_BUNDLE_DEPS Framework)
```

Comment:
I'm surprised that configadmin isn't in the `US_BUNDLES` list. this looks like an oversight and I don't think it needs to be addressed in this PR. Can you create a separate issue to capture fixing this oversight? Thanks.

Reply:
Yes, [Done](https://github.com/CppMicroServices/CppMicroServices/issues/1101).